jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ToolkitAuthManager.kt

```diff
@@ -204,11 +204,14 @@ internal fun reauthConnection(project: Project?, connection: ToolkitConnection):
     return provider
 }
 
+@Suppress("UnusedParameter")
 fun logoutFromSsoConnection(project: Project?, connection: AwsBearerTokenConnection, callback: () -> Unit = {}) {
     try {
-        ApplicationManager.getApplication().messageBus.syncPublisher(BearerTokenProviderListener.TOPIC).invalidate(connection.id)
         ToolkitAuthManager.getInstance().deleteConnection(connection.id)
-        project?.let { ToolkitConnectionManager.getInstance(it).switchConnection(null) }
```

Comment:
i feel like this line was needed for an UI issue

Reply:
When the auth manager deletes a connection, it sends a message via `BearerTokenProviderListener.TOPIC` which is caught by the connection manager. The connection manager then calls `switchConnection(null)`.